cmd/clusterlink/cmd/deploy/deploy_peer.go

```diff
@@ -185,6 +188,7 @@ func (o *PeerOptions) Run() error {
 		FabricCertificate:       fabricCert,
 		PeerCertificate:         peerCert,
 		CACertificate:           caCert,
+		Controlplanes:           o.ControlplaneReplicas,
```

Comment:
if we maintain backward compatibility by defining that if replicas is not set at all (i.e., the zero value for uint16 is set by default) we treat it as if it was set to `1` - we can save changes across multiple files.
Somewhere we can say:
```go 
   ...
   if controlplaneReplicas == 0 { // not set, use default replica count
      controlplaneReplicas = 1
   }
```